pygmt/datasets/samples.py

```diff
@@ -255,16 +245,10 @@ def load_fractures_compilation(**kwargs):
         columns.
```

Comment:
Like the load_japan_quakes function, maybe we should say the returned column names are xxxx and xxxx.

Reply:
Done!